docs/src/main/sphinx/admin/properties-write-partitioning.rst

```diff
@@ -0,0 +1,26 @@
+=============================
+Write partitioning properties
+=============================
+
+``use-preferred-write-partitioning``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Enable preferred write partitioning. When set to ``true`` and more than the
+minimum number of partitions, set in ``preferred-write-partitioning-min-number-of-partitions``,
+are written, each partition is written by a separate writer. As a result, for some connectors such as the
+Hive connector, only a single new file is written per partition, instead of
+multiple files. Partition writer assignments are distributed across worker
+nodes for parallel processing.
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of written partitions that is required to use connector
```

Comment:
`The minimum number of written partitions that is required...`
That makes it sound like this is based on the actual number of written partitions, but it's based on an estimate of number of partitions derived from statistics.
Maybe
`Use connector ``preferred write partitioning`` when the optimizer's estimate of the number of partitions that can be written by a query is greater than the provided value`

Reply:
I completely agree, please do send a PR to clean this up!

(I didn't want to mix in content changes with the move)

Reply:
Created https://github.com/trinodb/trino/pull/7916 with the suggested changes here.